documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+```
+#### Game Durations - monthly
+You can use the ***Nether.Analytics.GetGameDurationMonthly*** method to get the average game duration per month. Returns an arry of Duration
+##### Example usage
+```javascript
+var durations = Nether.Analytics.GetGameDurationMonthly();
+console.log(durations[0].AverageDuration + ' for ' + durations[0].Month + '-' + durations[0].Year);
+```
+#### Game Durations - yearly
+You can use the ***Nether.Analytics.GetGameDurationYearly*** method to get the average game duration per year. Returns an arry of Duration
+##### Example usage
+```javascript
+var durations = Nether.Analytics.GetGameDurationYearly();
+console.log(durations[0].AverageDuration + ' for ' + durations[0].Year);
+```
+#### Level Drop Offs - daily
+You can use the ***Nether.Analytics.GetLevelDropOffsDaily*** method to get the total count of level drop offs by level and day. Returns an arry of DropOff
+##### Example usage
+```javascript
+var dropOffs = Nether.Analytics.GetLevelDropOffsDaily();
+console.log('Level ' + dropOffs[0].ReachedLevel + 'Drop offs: ' + dropOffs[0].TotalCount + ' for ' + dropOffs[0].Day + '-' + dropOffs[0].Month + '-' + dropOffs[0].Year);
+```
+#### Level Drop Offs - monthly
+You can use the ***Nether.Analytics.GetLevelDropOffsMonthly*** method to get the total count of level drop offs by level and month. Returns an arry of DropOff
+##### Example usage
+```javascript
+var dropOffs = Nether.Analytics.GetLevelDropOffsMonthly();
+console.log('Level ' + dropOffs[0].ReachedLevel + 'Drop offs: ' + dropOffs[0].TotalCount + ' for ' + dropOffs[0].Month + '-' + dropOffs[0].Year);
+```
+#### Level Drop Offs - yearly
+You can use the ***Nether.Analytics.GetLevelDropOffsYearly*** method to get the total count of level drop offs by level and year. Returns an arry of DropOff
+##### Example usage
+```javascript
+var dropOffs = Nether.Analytics.GetLevelDropOffsYearly();
+console.log('Level ' + dropOffs[0].ReachedLevel + 'Drop offs: ' + dropOffs[0].TotalCount + ' for ' + dropOffs[0].Year);
+```
+### Identiy
+
+### Leaderboard
+
+#### Get Leaderboard
+You can use the ***Nether.Leaderboard.GetLeaderboard*** method to get the currently configured Leaderboard. Returns an array of Leaderboard
+##### Example usage
+```javascript
+var leaderboard = Nether.Leaderboard.GetLeaderboard();
+```
+### Get Leaderboard by name
+You can use the ***Nether.Leaderboard.GetLeaderboardFromName*** method to get all of the gammer tags and highscores for the specified Leaderboard. Returns a LeaderboardSummary object
+#### Example usage
+```javascript
+var leaderboardScores = Nether.Leaderboard.GetLeaderboard('testboard');
+```
+
+### Put score
+You can use the ***Nether.Score.PutScore*** method to add a players score.
+#### Example usage
+```javascript
+Nether.Scores.AddScore('UK', 100, 'customTag');
+Nether.Scores.AddScore('UK', 100);
+```
+
+### Drop scores
```

Comment:
@stuartleeks I'm assuming this would be used for "clear my scores" functionality in a game?